packages/shared/lib/sdk/sync.ts

```diff
@@ -487,6 +488,13 @@ export class NangoAction {
                 };
             }
         }
+        if (!config.axios?.response) {
+            axiosSettings.interceptors = {
+                response: {
+                    onFulfilled: this.logAPICall.bind(this)
+                }
+            };
+        }
```

Comment:
do I read correctly that it doesn't override the interceptors defined a few lines above that is used for snapshoting in dry-mode?

Reply:
yes exactly, I'll add a comment